Mathlib/RingTheory/GradedAlgebra/Basic.lean

```diff
@@ -352,3 +377,34 @@ theorem coe_decompose_mul_of_right_mem (n) [Decidable (i ≤ n)] (b_mem : b ∈
 end DirectSum
 
 end CanonicalOrder
+
+section GradeZero
+
+variable {ι A σ : Type*}
+variable [Ring A]
+variable [DecidableEq ι] [CanonicallyOrderedAddCommMonoid ι]
+variable [SetLike σ A] [AddSubgroupClass σ A]
+variable (𝒜 : ι → σ) [GradedRing 𝒜]
+
+/--
+The ring homomorphism from `A` to `𝒜 0` sending every `a : A` to `a₀`.
+-/
+def GradedRing.projZeroRingHom'' : A →+* (𝒜 0) :=
+  ((GradedRing.projZeroRingHom 𝒜).codRestrict _ fun _x => SetLike.coe_mem _ :
+  A →+* GradeZero.subring 𝒜)
```

Comment:
There's no need to have this definition, its the same as the one further up.

Reply:
Yeah, it is unnecessary. I have deleted the definition and the whole section related to it.

Reply:
Don't forget to push!